src/server/ua_services_monitoreditem.c

```diff
@@ -402,6 +402,47 @@ Operation_CreateMonitoredItem(UA_Server *server, UA_Session *session,
                              newMon->parameters.samplingInterval,
                              (unsigned long)newMon->queueSize);
 
+     /* From this point on, after creating and activating the MonitoredItem,
+     * the server can access the EventFilter which is contained in the MonitoredItem.
+     * According to 7.17.3 in Part 4 : The Server shall validate the whereClause and the selectClauses when a Client creates or updates the EventFilter.
+     * The initial validation of the WhereClause and SelectClause are done here.
+     * EventFilterResult isn't currently being used to send Information to the Client.
+     * If the Validation is successful the Server prints a Notification and continues running,
+     * otherwise the connection with the Client will be terminated */
+#ifdef UA_ENABLE_SUBSCRIPTIONS_EVENTS
+    if(newMon->parameters.filter.content.decoded.type == &UA_TYPES[UA_TYPES_EVENTFILTER]) {
+        UA_EventFilter *eventFilter =
+            (UA_EventFilter *)newMon->parameters.filter.content.decoded.data;
```

Comment:
Do we need a Null check here?

Reply:
No, a null check is not needed as far as we can tell. Why do you believe one might be needed?